include/PruManager.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * PruManager.h
+ *
+ * Support for interaction with PRU via
+ * (rproc+mmap) and/or (uio+libprussdrv)
+ *
+ *	Created on: Jul 3, 2021
+ *		Author: Dhruva Gole
+ */
+
+#include <string>
+
+#if ENABLE_PRU_UIO == 1
+#include <prussdrv.h>
+#endif
+
+#include <map>
+#include "Mmap.h"
+
+class PruManager
+{
+// expose parameters for the relevant paths
+public:
+	unsigned int pru_num, verbose;
```

Comment:
add a virtual destructor (compiler is complaining about it)

Reply:
done